README.md

```diff
@@ -628,6 +629,15 @@ export const Dialog = styled.dialog`
 `
 ```
 
+### Fonts
+First a few dos and don'ts:  
+- **DON'T** use fonts downloaded from a non-reliable source
+- **DON'T** use fonts converted by tools like Font Squirrel
```

Comment:
Maybe also state that this should also not be used as a temporarily solution. A lot of times I see people doing this because the licensed font will be added later. Later = forgotten

Reply:
@tworrisb gets it. Do not accept "oh I found this somewhere" font packages, even as a workaround. Violating font licenses can be legally and financially damaging things.